src/player.cpp

```diff
@@ -2016,7 +2016,8 @@ void player::memorial( std::ostream &memorial_file, std::string epitaph )
 
     //HP
 
-    const auto limb_hp = [this, &memorial_file, &indent]( const std::string &desc, const body_part bp ) {
+    const auto limb_hp = [this, &memorial_file, &indent]( const std::string & desc,
+    const body_part bp ) {
```

Comment:
what happened here? astyle starts the linebreak at the same column as the outer expression?